.github/workflows/packaging.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           path: ./dist
       - name: Install the prerequisites
         run: |
-          ${{ matrix.py_cmd }} -m pip install pip wheel
+          ${{ matrix.py_cmd }} -m pip install pip wheel jupyterlab~=3.0 notebook~=6.4
```

Comment:
Not sure how the extension checks could have worked before without these packages :thinking: